ipam/claim.go

```diff
@@ -12,6 +12,7 @@ type claim struct {
 	resultChan chan<- error
 	ident      string
 	addr       address.Address
+	reclaim    bool
 }
 
 func (c *claim) sendResult(result error) {
```

Comment:
As a drive-by comment: I find it somewhat counter-intuitive that `claim.sendResult(nil)` indicates a successful operation.